website/docs/faq.md

```diff
@@ -17,6 +17,20 @@ To patch vulnerabilities for applications, you can package these applications an
 
 If you find that your storage is rapidly being taken up after working with Copa, run `docker system prune`. This will prune all unused images, containers and caches. 
 
+## How does Copa determine what tooling image to use?
+
+All images being passed into Copa has their versioning data carefully stripped so that an appropriate tooling image can be obtained from a container repository.
+
+Debian: All debian images have their `minor.patch` versioning stripped and `-slim` appended. e.g. if `debain:12.6` is passed in, `debian:12-slim` is used as the tooling image.
+
+Ubuntu: All Ubuntu images use the same versioning that was passed in. e.g. if `ubuntu:24.04` is passed in, `ubuntu:24.04` will be used for the tooling image.
```

Comment:
were we able to confirm this with a test?

Reply:
I was able to on my machine. Here's the relevant code for that determination. If no scan is passed in, manifest is nil and we'll hit the else block for Ubuntu images to use Ubuntu tooling images. If the manifest is nil or it contains Debian, we default to Debian for tooling images.

![image](https://github.com/user-attachments/assets/342e93a8-df2a-4a26-a3bb-14c49576378f)

Reply:
it looks like we are setting osType to Debian always?